osu.Game.Rulesets.Osu/Mods/OsuModTransform.cs

```diff
@@ -27,26 +28,39 @@ internal class OsuModTransform : Mod, IApplicableToDrawableHitObjects
         public void ApplyToDrawableHitObjects(IEnumerable<DrawableHitObject> drawables)
         {
             foreach (var drawable in drawables)
+                drawable.ApplyCustomUpdateState += applyTransform;
+        }
+
+        private void applyTransform(DrawableHitObject drawable, ArmedState state)
+        {
+            switch (drawable)
             {
-                var hitObject = (OsuHitObject)drawable.HitObject;
+                case DrawableSliderHead _:
+                case DrawableSliderTail _:
+                case DrawableSliderTick _:
+                    return;
```

Comment:
Should also include `DrawableRepeatPoint` here.

Reply:
good catch. that's actually the one i was trying to fix with these exceptions, too...